core/chaincode/shim/chaincode.go

```diff
@@ -148,40 +147,20 @@ func getPeerAddress() string {
 	}
 
 	if peerAddress = viper.GetString("peer.address"); peerAddress == "" {
+		os.Exit(-1)
```

Comment:
This strikes me as something that probably warrants a log/stderr message before exiting.  IIUC, we are basically trying to say that omitting peer.address is fatal, right?